SU-SWS/stanford_news
# D8CORE-2292: Added in the More news button

# READY FOR REVIEW

# Summary
- added the See More News to the vertical cards.

# Needed By (Date)
- Thursday

# Urgency
- Medium

# Steps to Test

1. Pull in this change
2. Get the configuration change from profile.
3. Refresh the css.
4. Verify on the sample news article at the bottom o

# Affected Projects or Products
- D8Core-2292 in News module and in the Profile module.

# Associated Issues and/or People
- D8Core-2292
- https://github.com/SU-SWS/stanford_news/pull/87
- https://github.com/SU-SWS/stanford_profile/pull/239

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
